### PR TITLE
[ci] Add Ubuntu 24.10 Build

### PR DESCRIPTION
**Pull request by dpiparo**
Build clad with one core only.

### CODE REVIEW

File: interpreter/cling/tools/plugins/clad/CMakeLists.txt [hunk 1]
```diff
@@ -95,8 +95,8 @@ ExternalProject_Add(
              -DLLVM_DIR=${LLVM_BINARY_DIR}
              -DCLANG_INCLUDE_DIRS=${CLANG_INCLUDE_DIRS}
              ${_clad_extra_cmake_args}
-  BUILD_COMMAND ${CMAKE_COMMAND} --build . ${EXTRA_BUILD_ARGS}
-  INSTALL_COMMAND ${CMAKE_COMMAND} --build . ${EXTRA_BUILD_ARGS} --target install
+  BUILD_COMMAND ${CMAKE_COMMAND} --build . ${EXTRA_BUILD_ARGS} -j 1
```
Comment (pcanal) [inline interpreter/cling/tools/plugins/clad/CMakeLists.txt:98]: Consider adding a comment why '-j 1' is needed (is it a temporary workaround until something is fixed in clad or is it a limitation of the platform and/or MV configuration?).

Comment (dpiparo) [reply interpreter/cling/tools/plugins/clad/CMakeLists.txt:98]: I agree. I wanted to test this at scale before documenting further. @vgvassilev any thought about why this is now necessary?

Comment (vgvassilev) [reply interpreter/cling/tools/plugins/clad/CMakeLists.txt:98]: I am not sure I have seen the actual build failure to be able to comment. Can you give me a link to the failure?

Comment (dpiparo) [reply interpreter/cling/tools/plugins/clad/CMakeLists.txt:98]: Just remove the "-j1"

Comment (vgvassilev) [reply interpreter/cling/tools/plugins/clad/CMakeLists.txt:102]: I do not have access to ubu24 system.

Comment (dpiparo) [reply interpreter/cling/tools/plugins/clad/CMakeLists.txt:102]: but you have the line to reproduce the failure locally with the container, right?

**Comment by guitargeek** [comment]
I just can't reproduce and debug these failues on a clean ubuntu 24.10 install... I don't think we can make progress on this in the next 10 day :(

Would it make sense maybe to add it to the CI with `pyroot=OFF`? At least we get some coverage then (and also test a new point in the configuration phase space).

File: interpreter/cling/tools/plugins/clad/CMakeLists.txt (hunk 1 above)
Comment (andresailer) [reply interpreter/cling/tools/plugins/clad/CMakeLists.txt:102]: https://github.com/root-project/root/issues/16733#issuecomment-2435180193

**Comment by vepadulano** [comment]
This PR is fixing the ubuntu 24.10 failures https://github.com/root-project/root/pull/16837 . We can introduce the new CI build directly with that.

**Pull request closed without merging**